### ISSUE TITLE
--include does not work when in 'opts' file [Bug]

### ISSUE THREAD

**Issue by gdarai** [sysgears/mochapack#73]
- [ ] I'd be willing to submit the fix

**Describe the bug**
I am upgrading our libraries, so I swapped to mochapack and now I have an issue:
We are having global memory clearing method in cleanUpResources.js

```
afterEach('Cleanup after test', function () {
    // Remove all listeners added by our test
    console.log('-- clear --');
    listeners.window.forEach((listener) => window.removeEventListener(listener.type, listener.listener));
    listeners.document.forEach((listener) => document.removeEventListener(listener.type, listener.listener));
    listeners.window = [];
    listeners.document = [];
    // Restore sinon sandbox
    sinon.reset();
    sinon.restore();
});
```
Loading this file using
```  --include ./test/cleanupResources.js ```
also using parameter
``` --opts mochapack.opts ```

```
--colors
--recursive
--full-trace
--require mock-local-storage
--require ./test/test_helpers.js
--include ./test/cleanupResources.js
```
clear and concise description of what the bug is.

when I have the ``` include ``` inline directly it loads the cleanup file. When I have it in the opts file, it does not.

**To Reproduce**

You can do the same thing I did... have an --include with some file and in it have afterEach with console.log and check the difference with inline and in opts solution.

**Environment if relevant (please complete the following information):**

 - OS: Linux
 - Node version 10.18
├── mocha@7.2.0 
├── mochapack@2.0.0 
└── webpack@4.43.0 

Currently I am bypassing this using the inline solution.

**Comment by panta82** [comment]
![image](https://user-images.githubusercontent.com/4291324/90819612-7c34ec80-e330-11ea-88f4-82a8389a3033.png)

It seems opts file is only parsed if you specify --opts arg. It no longer loads the file by default.

This behavior is a deal breaker for me. I am using opt-file to point the correct webpack config when I run random tests from my IDE. Without it, I'd have to edit every run configuration manually. Too fiddly.

**Comment by Jack-Barry** [comment]
Looks like I may have forgotten to include the opts file path [here](https://github.com/sysgears/mochapack/blob/556836c27e8e5e7d49b2feb515ba16c0a7a69620/src/cli/argsParser/optionsFromParsedArgs/mocha/mochaOptionsFromParsedArgs.ts#L137-L140).

Just needs `parsedMochaArgs.opts` added as the third argument as far as I can tell. 

@gdarai Would you be able to test this in your project to see if it fixes what you're getting here?

@panta82 The `opts` file is deprecated by Mocha, so automatically using it is no longer the strategy used/supported by Mochapack. When you say you're pointing to Webpack config from it, pardon my ignorance but I'm not sure I catch the difference between updating a CLI script argument and updating the contents of the `opts` file to accomplish this, in both cases don't you still have a manual step involved to configure?

**Comment by panta82** [comment]
@Jack-Barry The trick is that Webstorm IDE allows me to click a little "play" button next to unit tests and run them immediately in an ad-hoc fashion. And I couldn't figure out how to tell Webstorm to use a particular CLI arg when launching tests that way. So an opts file worked nicely for that.

**Comment by Jack-Barry** [comment]
@panta82 Would you be able to provide those arguments using a custom run configuration as outlined [here](https://www.jetbrains.com/help/webstorm/creating-and-editing-run-debug-configurations.html)? I haven't used Webstorm but from those docs it looks like you should be able to customize the script triggered by the play button.

**Comment by panta82** [comment]
@Jack-Barry Yes, but this is a custom run configuration. You would create these for permanent things you would want to run all the time. Eg. "All tests", "All unit tests", etc.

The problem is when I just want to run an ad-hoc individual test while working on a new code or debugging something. Sure I could click, see it crash, then edit the created run configuration, but that's way more fiddly than just using the version of mocha that supports .opts. :)

**Comment by dhartjes** [comment]
Hello! I'm wondering if the opts section could be removed from the documentation. Thanks for picking this up!

**Comment by ascii-soup** [comment]
@panta82 Forgive me if you are aware of this already, I appreciate this is months later, but you can edit the templates for ad-hoc run configurations in Webstorm - just look for the 'Templates' section in the list below your normal run configurations. Here you can select 'Mocha' for example, and then set the properties you wish to be inherited by _all new run configurations_ created for Mocha inside that project. Hope this helps!

**Comment by jcjp** [comment]
What's the alternative for the `*.opts` file? Just add it on the command itself?

**Comment by Bistard** [comment]
@jcjp My command `mochapack --opts /path/to/mochapack.opts` won't work somehow, the configuration will not be passed into mochapack.

My alternatives are: write your configuration in any json file, and write another script say `unit-test.js`, this script simply reads the configurations and build a command in string then run another process to execute that command in string.
